scylla/README.md

```diff
@@ -0,0 +1,96 @@
+<!--
+Copyright (c) 2015 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+# Scylla CQL binding
+
+Binding for [Scylla](https://www.scylladb.com/), using the CQL API
+via the [Scylla
+driver](https://github.com/scylladb/java-driver/).
+
+## Creating a table for use with YCSB
+
+For keyspace `ycsb`, table `usertable`:
+
+    cqlsh> create keyspace ycsb
+        WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3 };
+    cqlsh> USE ycsb;
+    cqlsh> create table usertable (
+        y_id varchar primary key,
+        field0 varchar,
+        field1 varchar,
+        field2 varchar,
+        field3 varchar,
+        field4 varchar,
+        field5 varchar,
+        field6 varchar,
+        field7 varchar,
+        field8 varchar,
+        field9 varchar);
+
+**Note that `replication_factor` and consistency levels (below) will affect performance.**
+
+## scylla Configuration Parameters
+
+- `hosts` (**required**)
+  - scylla nodes to connect to.
+  - No default.
+
+* `port`
+  * CQL port for communicating with scylla cluster.
```

Comment:
Please scope this config, e.g. `scylla.port`

If the port must be the same for all hosts (which is implicit here) please expressly say that.

Reply:
done